src/components/inputs/TextField.js

```diff
@@ -642,6 +679,19 @@ function createStyles({placeholderTextColor, centered, multiline}) {
     label: {
       ...LABEL_TYPOGRAPHY,
       height: LABEL_TYPOGRAPHY.lineHeight
+    },
+    rightIcon: {
+      alignSelf: 'center', 
+      marginLeft: 8
+    },
+    rightButton: {
+      position: 'absolute',
```

Comment:
I don't think the alignment in the example looks good, maybe, it's worth checking with Yulia if that what she really wants. 

Anyway, even If I don't use floatingPlaceholder, I would expect the button/image to be aligned nicely to the text, but it's not.

Reply:
I told her that and asked her if that what's she wants. She said yes